src/gui/main-styles.ts

```diff
@@ -554,6 +554,13 @@ styles.registerStyle("main", () => {
 			"min-width": 0,
 			"white-space": "nowrap",
 		},
+		".text-ellipsis-multi-line": {
+			display: "-webkit-box",
```

Comment:
oh, today I've learned an obscure CSS trick, thank you!

perhaps we could add some info on it? I have found that it's mentioned here and despite being prefixed it is actually standardized and widely supported
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp